config.go

```diff
@@ -616,6 +618,20 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// If RelayTxFeeRate is non-default we apply it over cfg.relayTxFeeRate
+	// However if it is default we shouldn't because its possible
+	// that the deprecated 'minRelayTxFee' is being used.
+	if cfg.RelayTxFeeRate != defaultRelayTxFeeRate.ToBTC() {
```

Comment:
Rather than repeating all of this, I think it would be cleaner to prevent both from being set at the same time (before the previous validation section), then set `cfg.RelayTxFeeRate = cfg.MinRelayTxFee` if `cfg.MinRelayTxFee` is not the default (i.e it has been set).  That way only a single parse/validate is needed.

In any case, both should not be allowed to be set simultaneously since they conflict.